application/src/main/java/uk/nhs/hee/tis/revalidation/repository/DoctorsForDBRepository.java

```diff
@@ -39,12 +39,14 @@
 
   @Query(value = "{ '$and' : [{ '$or' : [{'doctorFirstName' : { '$regex' : ?0, '$options' : 'i'}}, "
       + "{ 'doctorLastName' : { '$regex' : ?0, '$options' : 'i'}}, { '_id' : { '$regex' : ?0, '$options' : 'i'}}]}, "
-      + "{ 'designatedBodyCode' : { '$in' : ?1 }}, { 'underNotice' : { '$in' : ?2 }}]}")
+      + "{ 'designatedBodyCode' : { '$in' : ?1 }}, { 'underNotice' : { '$in' : ?2 }},"
+      + "{ 'existsInGmc' : 'true' }]}")
   Page<DoctorsForDB> findByUnderNotice(final Pageable pageable, final String searchQuery,
       final List<String> dbcs, final UnderNotice... underNotice);
 
   @Query(value = "{ '$and' : [{ '$or' : [{'doctorFirstName' : { '$regex' : ?0, '$options' : 'i'}}, "
-      + "{ 'doctorLastName' : { '$regex' : ?0, '$options' : 'i'}}, { '_id' : { '$regex' : ?0, '$options' : 'i'}}]}, { 'designatedBodyCode' : { '$in' : ?1 }}, { '_id' : { '$nin' : ?2 }}]}")
+      + "{ 'doctorLastName' : { '$regex' : ?0, '$options' : 'i'}}, { '_id' : { '$regex' : ?0, '$options' : 'i'}}]}, { 'designatedBodyCode' : { '$in' : ?1 }}, { '_id' : { '$nin' : ?2 }},"
```

Comment:
I'm happy if you don't but you could split this line just before `{ 'designatedBodyCode'...` to be more like the query above.

Reply:
will do!